app/main.py

```diff
@@ -1,6 +1,22 @@
+from functools import wraps
 from typing import Callable
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    store = {}
+
+    @wraps(func)
+    def wrapped(*args, **kwargs) -> dict:
+
+        key = (args, frozenset(kwargs.items()))
```

Comment:
include function name as well

Reply:
key = (func.__name__, args, frozenset(kwargs.items())) like this?